azure_monitor/src/azure_monitor/trace.py

```diff
@@ -115,22 +119,22 @@ def span_to_envelope(self, span):  # noqa pylint: disable=too-many-branches
                 data.properties["request.url"] = span.attributes["http.url"]
             if "http.status_code" in span.attributes:
                 status_code = span.attributes["http.status_code"]
-                data.responseCode = str(status_code)
+                data.response_code = str(status_code)
                 data.success = 200 <= status_code < 400
-            elif span.status == StatusCanonicalCode.OK:
+            elif span.status.canonical_code == StatusCanonicalCode.OK:
```

Comment:
Remove this condition and assignment, not needed after your change to set this up during "data" creation

Reply:
Done!